qiskit/scheduler/__init__.py

```diff
@@ -19,13 +19,22 @@
 
 A circuit scheduler compiles a circuit program to a pulse program.
 
+Core API
+========
+
 .. autoclass:: ScheduleConfig
 
 .. currentmodule:: qiskit.scheduler.schedule_circuit
 .. autofunction:: schedule_circuit
 .. currentmodule:: qiskit.scheduler
 
-.. automodule:: qiskit.scheduler.methods
```

Comment:
I inlined the module docstring and added a new header. Before, it was paragraph text and looked weird.